tests/report/reportportal/data/plan.fmf

```diff
@@ -3,7 +3,7 @@ summary: Testing the integration of tmt and Report Portal via its API
 discover:
     how: fmf
 provision:
-    how: local
+    how: container
```

Comment:
What is the reason for this change?

Reply:
The reportportal tests are not designed to run in the GH CI. Instead, they should be executed locally by a developer against a specific ReportPortal instance. That's why I think it's preferable to run them in a container or using the '-h virtual' option. I am not "--feeling-safe" enough to run them locally :).